src/Components/OptionCardGroup/OptionCardGroup.css

```diff
@@ -43,11 +57,17 @@
 
   div.sub-header,
   h2.sub-header,
-  .question-label.household-data-q-underline {
+  .question-label {
     font-size: 1.5rem;
   }
 
   p.question-label {
     font-size: 1rem;
   }
+
+  .checkmark {
+    height: 1.5rem;
+    width: 1.5rem;
+    margin: -2rem 0.5rem 0.5rem 0;
```

Comment:
The negative rem value allows it to show up without pushing the container up 2 rems.